Sources/Build/BuildPlan.swift

```diff
@@ -25,6 +25,18 @@ extension String {
     }
 }
 
+extension String {
```

Comment:
fells like this should be extensions on AbsolutePath rather than on String

Reply:
As a local extension to `AbsolutePath` is what you have in mind?

Reply:
right, this basically operates on the string version of absolute path, so the call sites are `<path>.pathString.nativePathString()` and would be better as  `<path>.nativePathString()`

Reply:
```
extension AbsolutePath {
  internal func nativePathString() -> String {
       return URL(fileURLWithPath: self.pathString).withUnsafeFileSystemRepresentation {
           String(cString: $0!)
       }
   }
}
```

Reply:
Agreed, just wanted to make sure that the location was still within this file :)

Reply:
also consider computed var vs. func to make it more like `pathString`, tho I can go both ways given the allocations involved

Reply:
one more suggestion: make it into `nativePathString(escaped: Bool)` so this way we can collapse these two into one clean API